middleware/logger/logger_test.go

```diff
@@ -147,7 +147,7 @@ type fakeErrorOutput int
 
 func (o *fakeErrorOutput) Write([]byte) (int, error) {
 	*o++
-	return 0, errors.New("fake output")
```

Comment:
I think this was intended to return an error (as opposed to type fakeOutput around line 630).

Reply:
sure